RocketChat/Rocket.Chat.ReactNative
# [FIX] Black screen during Share

## Proposed changes
Something about RNSplashScreen.hide is causing an issue here. I wrapped the call in a try/catch.

## Issue(s)	
#3088

## How to test or reproduce
Try to share an image either from within the app or from an another application to the app. After investigation, I have found this issue appears only when the server has the mobile screen lock value enabled. If it is disabled, sharing will work fine. If enabled, the call to RNSplashScreen.hide will cause a black screen to appear, and the user is unable to proceed.

## Screenshots

## Types of changes

- [ x] Bugfix (non-breaking change which fixes an issue)

## Checklist

- [x ] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [ x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [ x] Lint and unit tests pass locally with my changes

## Further comments
N/A
